src/Microsoft.AspNet.Mvc.ViewFeatures/ViewResult.cs

```diff
@@ -51,9 +51,9 @@ public class ViewResult : ActionResult
         public IViewEngine ViewEngine { get; set; }
 
         /// <summary>
-        /// Gets or sets the <see cref="MediaTypeHeaderValue"/> representing the Content-Type header of the response.
+        /// Gets or sets the media type representing the Content-Type header of the response.
         /// </summary>
```

Comment:
`the media type representing the Content-Type header of the response`

I would just say "the Content-Type header for the response"